src/client/src/AppTheme.ts

```diff
@@ -28,6 +28,8 @@ export const theme = createTheme({
       main: "#0b7285",
       secondary: "#a65462",
     },
+    boxShadow: "rgba(0, 0, 0, 0.17)",
+    background: { default: "#ffffff", lightgrey: "#f1f3f5", darkgrey: "#787878", dark: "rgba(0, 0, 0, 0.5)" },
```

Comment:
PP: Ich finde _lightgrey_, _darkgrey_ etc. schlechte namen, ich fände es besser, wenn die Farben nach ihrer Funktion im UI benannt werden, irgenwie _highlight_ oder so.

Reply:
Stimmt, finde es auch nicht ideal,  wir sind aber leider sehr inkonsistent bisher mit dem  Einsatz von Farben und Style im allgemeinen, sodass es schwer ist in diesem konkreten Fall für den Background stimmige Namen zu finden, die von der Funktion abgeleitet sind. Ich lasse es mal so, aber sollten wir denke ich weiter dranbleiben das zu optimieren.